docs/get-started.md

```diff
@@ -1,189 +1,73 @@
 # Get started
 
-To get started with Firebase Genkit, install the Genkit CLI and run
-`genkit init` in a Node.js project. The rest of this page shows you how.
+This guide shows you how to get started with Genkit in a Node.js app.
 
-## Requirements
+## Prerequisites
 
-Node.js 20 or later.
+This guide assumes that you're familiar with building applications with Node.js.
 
-Recommendation: The [`nvm`](https://github.com/nvm-sh/nvm) and
-[`nvm-windows`](https://github.com/coreybutler/nvm-windows) tools are a
-convenient way to install Node.
+To complete this quickstart, make sure that your development environment meets the following requirements:
 
-## Install Genkit {:#install}
+* Node.js v20+  
+* npm
 
-Install the Genkit CLI by running the following command:
+## Install Genkit dependencies
 
-```posix-terminal
-npm i -g genkit
-```
-
-This command installs the Genkit CLI into your Node installation directory
-so that it can be used outside of a Node project.
+Install the following Genkit dependencies to use Genkit in your project:
 
-## Create and explore a sample project {:#explore}
+* `@genkit-ai/ai` and `@genkit-ai/core` provide Genkit core capabilities. 
+* `@genkit-ai/googleai` provide access to the Google AI Gemini models.
+* `genkit` provides the Genkit CLI and tooling to help you test and debug your solution later.
 
-1.  Create a new Node project:
-
-    ```posix-terminal
-    mkdir genkit-intro && cd genkit-intro
+```posix-terminal
+npm install @genkit-ai/ai @genkit-ai/core @genkit-ai/googleai
+npm install -g genkit
```

Comment:
```suggestion

npm install -g genkit
```

Reply:
Is the change just adding an additional line between the terminal commands?